### ISSUE TITLE
Importing css causes obscure error in browserify

### ISSUE THREAD

**Issue by udalrich** [Lighthouse-io/react-visjs-timeline#37]
# Overview

When I attempt to import `react-visjs-timeline` and then transform my source with `browserify`, it says "Error runing browserify Unexpected token undefined".  

## Package versions
react-visjs-timeline: 1.4.0
vis version: 4.21.0
browserify version: 13.3.0

## Steps to reproduce

1. Extract the files from the attached archive. 
2. `npm install`
3. `npm install -g gulp` if not already installed
3. `gulp`

### Output

    $ gulp
    [15:14:01] Using gulpfile ~/test/gulp/gulpfile.js
    [15:14:01] Starting 'default'...
    [15:14:01] Starting 'browserify'...
    [15:14:01] Finished 'browserify' after 1.2 ms
    [15:14:01] Finished 'default' after 2.39 ms

    events.js:163
          throw er; // Unhandled 'error' event
          ^
    SyntaxError: Unexpected token

If I remove the `require(... .css)` line from `node_modules/react-visjs-timeline/build/index.js`, then the output is what I expect.

    $ gulp
    [15:16:38] Using gulpfile ~/test/gulp/gulpfile.js
    [15:16:38] Starting 'default'...
    [15:16:38] Starting 'browserify'...
    [15:16:38] Finished 'browserify' after 1.23 ms
    [15:16:38] Finished 'default' after 2.19 ms
    Ran browserify and wrote output

I am using `browserify-css` which seems like it should handle the import of css, but it is clearly not doing that.

[reproduce.zip](https://github.com/Lighthouse-io/react-visjs-timeline/files/1380395/reproduce.zip)

**Comment by udalrich** [comment]
One possible resolution is to document how to import CSS into the JSX file with babel and browserify.

**Comment by educastellano** [comment]
I think the CSS should be imported outside the component, so the user can import it in whatever bundler is using:

```js
import Timeline from 'react-visjs-timeline'
import 'vis/dist/vis.min.css'

ReactDOM.render(
  <Timeline />,
  document.getElementById('root')
);

```

**Comment by jdehaan** [comment]
Thanks for describing so nicely the issue. I run in a very similar scenario with webpack. I also think the culprit is the inlined css removing manually `require('vis/dist/vis-timeline-graph2d.min.css');` from the index.js and rebuilding makes things load without cryptic error messages about missing loaders...

**Comment by willmcclellan** [comment]
I guess browserify needs to be told how load the css correctly? I haven't used browserify in years so can't really offer any insight. Would need someone to submit a PR to get and update :)

**Comment by zoeesilcock** [comment]
@educastellano I agree that the component should not contain a require to the CSS file from the vis package. It makes the assumption that the build environment knows how to handle CSS requires inside of node_modules. I would very much prefer having a way to import just the component without the CSS so I can handle that in what would be a suitable way for my project.

@jdehaan Did you find a solution when using Webpack? I've tried in all sorts of ways, but I can't get Webpack to handle the fact that it's trying to require the CSS when doing server side rendering. I get the feeling that this is by design as npm packages are supposed to be self contained and not require this type of transformation.

**Comment by karangarg45** [comment]
Anyone found a solution for webpack?
even i am getting Cannot read property call of undefined with Webpack 3

**Comment by metaheap** [comment]
> Thanks for describing so nicely the issue. I run in a very similar scenario with webpack. I also think the culprit is the inlined css removing manually `require('vis/dist/vis-timeline-graph2d.min.css');` from the index.js and rebuilding makes things load without cryptic error messages about missing loaders...

I had to do this to get it working with next.js.
